lib/bddsm/describe.rb

```diff
@@ -12,7 +12,8 @@ def run
       instance_eval(&@block)
     end
 
-    def it(&)
+    def it(comment = nil, &)
+      @comment = comment
```

Comment:
Are we going to have single comment to all 'its' in a describe block? Looks like we should have separated

Reply:
Do you mean you need to comment out the remaining blocks?

Reply:
No. If we have multiple ìt`-s in a describe block, the last 'it' should override `@comment`, so the only one comment is going to be stored. 

```ruby
describe "test" do
  it "this one is going to be removed" do
  end

  it "this one will be saved" do
  end
end
```

Reply:
Ah, I understand. I'll fix it soon.